### DISCUSSION TITLE
how can I generate traffic when the canary is in paused at 50/50 for prometheus to acquire data?

### DISCUSSION THREAD

**Discussion by nilampatel-engineer** [argoproj/argo-rollouts#1947] [General]
## Summary 

What happened/what you expected to happen?

## Diagnostics

What version of Argo Rollouts are you running?

```
# Paste the logs from the rollout controller

# Logs for the entire controller:
kubectl logs -n argo-rollouts deployment/argo-rollouts

# Logs for a specific rollout:
kubectl logs -n argo-rollouts deployment/argo-rollouts | grep rollout=<ROLLOUTNAME>
```

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritize the issues with the most 👍.

**Comment by kostis-codefresh** [comment]
Hello

You have already asked this question 2 more times and I answered

https://github.com/argoproj/argo-rollouts/discussions/1944
https://github.com/argoproj/argo-rollouts/discussions/1954

Please don't open new discussions for the same topic
